RoslynPluginGenerator/DiagnosticAssemblyScanner.cs

```diff
@@ -53,7 +53,7 @@ public IEnumerable<DiagnosticAnalyzer> InstantiateDiagnostics(string directoryPa
         /// <see cref="DiagnosticAnalyzer"/>
         /// </summary>
         /// <returns>empty enumerable if no diagnostics were found</returns>
-        private IEnumerable<DiagnosticAnalyzer> InstantiateDiagnosticsFromAssembly(string assemblyPath, string language)
+        public IEnumerable<DiagnosticAnalyzer> InstantiateDiagnosticsFromAssembly(string assemblyPath, string language)
```

Comment:
YAGNI. We can change it to public later if it turns out to be useful.

FYI this is the sort of decision you should add a code review comment for to help reviewers understand why a change has been made (particularly since in this case you were reversing a change that was made in a recent pull request).